source/jormungandr/jormungandr/interfaces/v1/Places.py

```diff
@@ -86,19 +164,107 @@ def __init__(self, *args, **kwargs):
+                                            "value": "addr",
+                                            "boost": 1000
+                                        }
+                                    }
+                                },
+                                {
+                                    "match": {
+                                        "name.prefix": {
+                                            "query": q,
+                                            "boost": 100
+                                        }
+                                    }
+                                },
+                                {
+                                    "match": {
+                                        "name.ngram": {
+                                            "query": q,
+                                            "boost": 1
+                                        }
+                                    }
+                                },
+                                {
+                                    "function_score": {
+                                        "query": { "match_all": { } },
+                                        "field_value_factor": {
+                                            "field": "weight",
+                                            "modifier": "log1p",
+                                            "factor": 1
+                                        },
+                                        "boost_mode": "multiply",
+                                        "boost": 30
+                                    }
+                                }
+                            ]
+                        }
+                    },
+                    "filter": {
+                        "bool": {
+                            "should": [
+                                { "missing": { "field": "house_number" } },
+                                {
+                                    "query": {
+                                        "match": { "house_number": q }
+                                    }
+                                }
+                            ],
+                            "must": [
+                                {
+                                    "query": {
+                                        "match": {
+                                            "name.ngram": {
+                                                "query": q,
+                                                "minimum_should_match": "50%"
+                                            }
+                                        }
+                                    }
+                                }
+                            ]
+                        }
+                    }
+                }
+            }
+        }
+        try:
+            # TODO: get params?
+            es = Elasticsearch()
+            res = es.search(index="munin", size=args['count'], body=query)
```

Comment:
I don't understand where is the configuration.
How do we know which ES is called?